src/dh/README.md

```diff
@@ -0,0 +1,72 @@
+# Diffie-Hellman Key Exchanges
+
+## What is Diffie-Hellman?
+
+The Diffie-Hellman key exchange protocol, named after Whitfield Diffie and Martin Hellman, is a protocol for two parties to create a shared secret over a public channel without revealing information about the key.
+
+The exchange is made possible by using commutativity in finite cyclic groups, particularly with ellitpic curve cyclic groups. That is to say, given a common generator point on an ellitpic curve over a finite field $G \in E(\mathbb{F}_p)$, two parties, alice and bob may choose secrets, $a \in \mathbb{F}_p$ and $b \in \mathbb{F}_p$ respectively, compute their respective elliptic curve points $[a]G = A \in E(\mathbb{F}_p)$ and $[b]G = B \in E(\mathbb{F}_p)$ (again respectively), and exchange these on a public channel. Finally, they may perform the elliptic curve point arithmetic on their received points with their secrets as follows.
```

Comment:
Bit of advice that i found helpful when i was doing a lot of LaTeX on github. Make each new sentence on a new line!

With that said, there's a few typos:
ellitpic -> elliptic
alice -> Alice
bob -> Bob

Reply:
donestoyevsky